scripts/quests/hiddenQuests/A_Minstrel_In_Despair.lua

```diff
@@ -0,0 +1,47 @@
+-----------------------------------
+-- A Minstrel in Despair
+-----------------------------------
+-- Log ID: 3, Quest ID: 12
+-- Mertaire : !pos -17 0 -61 245
+-----------------------------------
+local quest = Quest:new(xi.questLog.JEUNO, xi.quest.id.jeuno.A_MINSTREL_IN_DESPAIR)
```

Comment:
This isn't a hidden quest, since it is tracked by a quest log entry. All the hidden quests DONT have a quest entry at any time, and are tracked by char vars (`local quest = HiddenQuest:new('TrustIngrid')` etc.).

This one is only slightly different in that it isn't visible until it's complete. Move it to the jeuno folder.

Presumably because it's only a single step and it doesn't contain `quest:begin` it should act like the wikis say

Reply:
That's a great point. It is in the right place on my implementation on hxi and asb - must have saw hidden and then put in hidden quest. Thanks, moved.